### PR TITLE
add batch processing to sherpa-onnx

### PR DESCRIPTION
**Pull request by jingzhaoou**

### CODE REVIEW

**Review by csukuangfj — COMMENTED** [review]
Could you also display the RTF at the end?

File: sherpa-onnx/csrc/sherpa-onnx.cc [hunk 1]
```diff
@@ -61,29 +69,28 @@ for a list of pre-trained models to download.
 
   sherpa_onnx::OnlineRecognizer recognizer(config);
 
-  float duration = 0;
+  std::vector<Stream> ss;
+  // std::vector<sherpa_onnx::OnlineStream *> p_ss;
```
Comment (csukuangfj) [inline sherpa-onnx/csrc/sherpa-onnx.cc]: Could you remove the commented code?

**Review by csukuangfj — COMMENTED** [review]
Thanks! Just left a minor comment.

File: sherpa-onnx/csrc/sherpa-onnx.cc (hunk 1 above)
Comment (jingzhaoou) [reply sherpa-onnx/csrc/sherpa-onnx.cc]: @csukuangfj  Just removed the commented code. Thanks.

**Comment by csukuangfj** [comment]
Thanks!

**Pull request merged**